packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -208,9 +212,9 @@ async function loginToRegistry(registry: containerDesktopAPI.Registry) {
 
   try {
     if (newRegistry) {
-      await window.createImageRegistry(registry.source, registry);
+      await window.createImageRegistry(registry.source, { ...registry });
```

Comment:
Sounds weird to me

Reply:
This is required or the hasBeenCalledWith assertion is going to fail in test. It happens because newRegistryForm is going to be cleaned of entered values right after calling window.createNewRegistry.